Formula/dar.rb

```diff
@@ -1,8 +1,8 @@
 class Dar < Formula
   desc "Backup directory tree and files"
   homepage "http://dar.linux.free.fr/doc/index.html"
-  url "https://downloads.sourceforge.net/project/dar/dar/2.6.10/dar-2.6.10.tar.gz"
-  sha256 "16a0b755c648c4ce6598e258ccf375f55d4f2f943448341ed41085ac7a27d2e7"
+  url "https://downloads.sourceforge.net/project/dar/dar/2.6.11/dar-2.6.11.tar.gz"
+  sha256 "5763e660b31ca494f67543026abfba778022915c8feef95672c9f5f5aa5cd4eb"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
https://github.com/Edrusb/DAR/blob/master/src/libdar/archive_aux.hpp#L5-L8